README.md

```diff
@@ -21,14 +21,44 @@ For DevOps or SRE teams this charm will make operating WordPress simple and stra
 It will allow easy deployment into multiple environments for testing of changes, 
 and supports scaling out for enterprise deployments.
 
+For information about how to deploy, integrate, and manage this charm, see the Official [wordpress-k8s-operator Documentation](https://charmhub.io/wordpress-k8s/docs).
+
+## Get Started
+
+To begin, refer to the [Getting Started](https://charmhub.io/wordpress-k8s/docs/tutorial) tutorial for step-by-step instructions.
+
+### Basic Operations
+
+The following actions are available for the charm - 
+- get-initial-password
+- rotate-wordpress-secrets
+
+You can find more information on this [here](https://charmhub.io/wordpress-k8s/actions)
+
+## Integrations
+
+Deployment of WordPress requires a relational database. The integration with the mysql interface is required by the wordpress-k8s charm for which `mysql-k8s` charm can be deployed as follows -
+
+```
+juju deploy mysql-k8s --trust
+# 'database' interface is required since mysql-k8s charm provides multiple compatible interfaces
+juju integrate wordpress-k8s mysql-k8s:database
+```
+
+Apart from this, the charm can be integrated with other Juju charms and services as well. You can find the full list of integrations [here](https://charmhub.io/wordpress-k8s/integrations)
+
+## Learn more
+
+- [Read more](https://charmhub.io/wordpress-k8s/docs)
+- [Official webpage](https://wordpress.com)
+
```

Comment:
I would add two more items in this list:

```
- [Developer documentation](https://codex.wordpress.org/Developer_Documentation)
- [Troubleshooting](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
```

I know that the Matrix channel is also linked below, but I think it's fine to include in the README twice.

Reply:
addressed